lib/services/role.go

```diff
@@ -493,6 +493,50 @@ func applyValueTraits(val string, traits map[string][]string) ([]string, error)
 	return interpolated, nil
 }
 
+// score is a sorting score of the rule, the larger the score, the more
```

Comment:
```suggestion
// scoreRule is a sorting score of the rule, the larger the score, the more
```
I'd actually name it - `ruleScore` as it's the rule's score and not the other way around